website/vercel.json

```
@@ -1037,6 +1037,38 @@
     {
       "source": "/docs/cq-vs-others/steampipe-alternative-cloudquery",
       "destination": "https://www.cloudquery.io/blog/steampipe-vs-cloudquery"
+    },
+    {
+      "source": "/docs/glossary/soc2",
+      "destination": "https://www.cloudquery.io/blog/soc2"
+    },
+    {
+      "source": "/docs/glossary/what-is-a-cspm",
+      "destination": "https://www.cloudquery.io/blog/what-is-a-cspm"
+    },
+    {
+      "source": "/docs/glossary/what-is-asm",
+      "destination": "https://www.cloudquery.io/blog/what-is-asm"
+    },
+    {
+      "source": "/docs/glossary/what-is-ciem",
+      "destination": "https://www.cloudquery.io/blog/what-is-ciem"
+    },
+    {
+      "source": "/docs/glossary/what-is-cnapp",
+      "destination": "https://www.cloudquery.io/blog/what-is-cnapp"
+    },
+    {
+      "source": "/docs/glossary/what-is-cwpp",
+      "destination": "https://www.cloudquery.io/blog/what-is-cwpp"
+    },
+    {
+      "source": "/docs/glossary/what-is-infrastructure-data-lake",
+      "destination": "https://www.cloudquery.io/blog/what-is-infrastructure-data-lake"
+    },
+    {
+      "source": "/docs/glossary/what-is-the-modern-data-stack",
+      "destination": "https://www.cloudquery.io/blog/what-is-the-modern-data-stack"
```

```suggestion
    },
    {
      "source": "/docs/glossary/overview",
      "destination": "https://www.cloudquery.io/blog"
    },
    {
      "source": "/docs/glossary/:path*",
      "destination": "https://www.cloudquery.io/blog/:path*"
```

I think this is missing the existing `/overview` URL https://docs.cloudquery.io/docs/glossary/overview, we can redirect it to the blog or even a specific tag if we need.

The rest can be done via a wildcard I think

---

Good call - thank you!